### ISSUE TITLE
Count vs Size method

### ISSUE THREAD

**Issue by billie66** [happypeter/haoduoshipin#139]
Here, there are two models named Episode and Comment with has_many association. That is, a episode has many comments, and a comment belong to a episode. So we can get all of the comments of the first episode named test, through the following:

``` ruby
test.comments
```

then how to know the number of these comments, use `count` or `size`?  However, we should know the difference of the two methods. 

If the first episode has ten comments, then 

``` ruby
test.comments.count  # the result is 10
test.comments.size    # the result is also 10
```

if delete two comments of the first episode, then

``` ruby
test.comments.count  # the result is 8
test.comments.size    # the result is still 10
```

In conclusion, count is proper method to evaluate the comments' number, which don't include empty records in comments table.
